docs/src/dataio_3_migration.rst

```diff
@@ -0,0 +1,293 @@
+fmu-dataio 3.0 migration guide
+================================
+
+This document contains a set of instructions on how to update your code to work
+with ``fmu-dataio`` 3.0. Almost all changes that have taken place are related to 
+fmu-dataio's ``ExportData`` class.
+
+
+ExportData
+-----------------------
+Changes to input arguments  
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
ExportData
----------
Changes to input arguments  
^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Sphinx usually issues warnings about these header indicators being longer than the text. You might want to double-check them all. (My suggestion here might be wrong since the font used for comments is not monospace)

Reply:
I got warnings if they were too short, so I made them extra long 😉 but I reset them too same length now, looks nicer 👍